snap/local/exporter-wrapper.bash

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+
+PROPERTIES=$(cat "${SNAP_COMMON}"/exporter.properties)
+BOOTSTRAP_SERVERS=$(echo "${PROPERTIES}" | grep bootstrap.servers | sed 's/bootstrap\.servers=//g')
+SECURITY_PROTOCOL=$(echo "${PROPERTIES}" | grep security.protocol | sed 's/security\.protocol=//g')
+SASL_JAAS_CONFIG=$(echo $PROPERTIES | grep sasl.jaas.config | sed 's/sasl\.jaas\.config=//g')
+PASSWORD=$(echo $SASL_JAAS_CONFIG | grep -oP "password\=\"([a-zA-Z0-9]+)" | cut -d "\"" -f 2)
+
+for i in $(echo $BOOTSTRAP_SERVERS | tr "," "\n")
+do
+    args+=("--kafka.server=${i}")
+done
+
+if [ -n "$(echo "${SECURITY_PROTOCOL}" | grep "SSL")" ]
+then
+    args+=" --tls.enabled"
+fi
+
+args+=" --sasl.enabled"
+args+=" --sasl.mechansim=scram-sha512"
+args+=" --sasl.username=exporter"
+args+=" --sasl.password=${PASSWORD}"
+args+=" --tls.ca-file=${SNAP_COMMON}/ca.pem"
+args+=" --tls.cert-file=${SNAP_COMMON}/server.pem"
+args+=" --tls.key-file=${SNAP_COMMON}/server.key"
```

Comment:
**question** Maybe this part should be within the if statement?

Reply:
Will do! I think it's safe to pass anyway but isn't very clear.